src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -0,0 +1,108 @@
+        return TypicalBlockModel.PATH_TO_CS2103T;
+    }
+
+    public static List<SuggestionItem> getExpectedSugForCs2103tPathInput() {
+        SuggestionItem cs2103t = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tLect = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tTut = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tTut1 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tTut2 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation(), () -> {});
+
+        List<SuggestionItem> expectedSuggestions;
+        SuggestionItem[] items = new SuggestionItem[]{cs2103t, cs2103tLect, cs2103tTut, cs2103tTut1, cs2103tTut2};
+        expectedSuggestions = List.of(items);
+
+        return expectedSuggestions;
+    }
+
+    public static List<String> getExpectedInputsForCs2103tPathInput(String userInputWithoutPath) {
+        List<String> expectedInputs = new ArrayList<>();
+
+        expectedInputs.add(userInputWithoutPath + TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath + TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation());
+
+        return expectedInputs;
+    }
+
+    /**
+     * Checks the correctness of the suggestions.
+     *
+     * @param expectedSuggestions The expected suggestions list.
+     * @param suggestions The actual list of suggestions.
+     */
+    public static void testSuggestions(List<SuggestionItem> expectedSuggestions, List<SuggestionItem> suggestions) {
+        assertEquals(expectedSuggestions.stream().map(s -> s.getProperty("displayText")).collect(Collectors.toList()),
+                suggestions.stream().map(s -> s.getProperty("displayText")).collect(Collectors.toList()));
+    }
+
+    /**
+     * Checks the correctness of the input stored in CommandInputModel for each suggestion.
+     *
+     * @param expectedInputs The expected list of inputs.
+     * @param suggestions The actual list of suggestions.
+     * @param model The app's model.
+     */
+    public static void testInputs(List<String> expectedInputs, List<SuggestionItem> suggestions, Model model) {
```

Comment:
I guess naming this as `assertInputs` is more intuitive right? That way it resembles the name of `JUnit` methods!